recipes/sqlalchemy-oso/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "sqlalchemy-oso" %}
+{% set build_number = 0 %}
+{% set version = "0.26.1" %}
+
+package:
+  name: {{ name|lower }}
```

Comment:
remove `name` variable: it only gets used here, and probably isn't going to change.